libcloud/compute/drivers/equinixmetal.py

```diff
@@ -491,7 +493,7 @@ def _to_size(self, data):
         except KeyError:
             cpus = None
         regions = [
-            region.get("href").replace("/metal/v1/facilities/", "")
+            region.get("href").replace("/metal/v1/metros/", "")
```

Comment:
```suggestion
            region.get("href").replace("/metal/v1/locations/metros/", "")
```

Reply:
I confirmed that `href` is returned with each metro and they follow the `/metal/v1/locations/metros/{uuid}` pattern.

When getting the plans list, if `?include=available_in_metros` is supplied, `href` will be omitted in the available_in_metros list of objects. These will contain id, name, code, country, and `price` (object with `hour` rate).

Reply:
Yes checked again
```json
"available_in": [
                {
                    "href": "/metal/v1/facilities/cbb790c1-812f-462a-8597-7229f35dc995",
                    "price": {
                        "hour": 8.25
                    }
                },
                {
                    "href": "/metal/v1/facilities/d37a89ba-5d74-4ea6-b30c-3464b4b20683",
                    "price": {
                        "hour": 7.5
                    }
                },
                {
                    "href": "/metal/v1/facilities/3576541d-3278-4bad-ba07-4a88ce986cad",
                    "price": {
                        "hour": 8.25
                    }
                },
                {
                    "href": "/metal/v1/facilities/4a84b014-90db-48a1-823b-c061430bf0c6",
                    "price": {
                        "hour": 8.25
                    }
                },
                {
                    "href": "/metal/v1/facilities/ec0d7b59-549b-4252-8177-f66a0da052a9",
                    "price": {
                        "hour": 9.0
                    }
                },
                {
                    "href": "/metal/v1/facilities/917e9941-8323-487f-8688-2b0b11baacd4",
                    "price": {
                        "hour": 7.5
                    }
                },
                {
                    "href": "/metal/v1/facilities/b96126f9-156a-43cd-a044-997082ec07eb",
                    "price": {
                        "hour": 9.0
                    }
                }
            ],
            "available_in_metros": [
                {
                    "href": "/metal/v1/locations/metros/108b2cfb-246b-45e3-885a-bf3e82fce1a0",
                    "price": {
                        "hour": 8.25
                    }
                },
                {
                    "href": "/metal/v1/locations/metros/96a57b6d-c62c-41b5-ab8e-f8d63a7f9887",
                    "price": {
                        "hour": 7.5
                    }
                },
                {
                    "href": "/metal/v1/locations/metros/b1ac82b2-616c-4405-9424-457ef6edf9ae",
                    "price": {
                        "hour": 8.25
                    }
                },
                {
                    "href": "/metal/v1/locations/metros/f6ada324-8226-4bfc-99a8-453d47caf2dc",
                    "price": {
                        "hour": 9.0
                    }
                },
                {
                    "href": "/metal/v1/locations/metros/2991b022-b8c4-497e-8db7-5a407c3a209b",
                    "price": {
                        "hour": 7.5
                    }
                },
                {
                    "href": "/metal/v1/locations/metros/d2f09853-a1aa-4b29-aa9d-682462fd8d1d",
                    "price": {
                        "hour": 9.0
                    }
                }
            ]
```